detail.scss

```diff
@@ -0,0 +1,51 @@
+.title {
+  width: 100%;
+  color: #000;
```

Comment:
Move the color to a SASS variable and use it instead of the hexa

Reply:
Done